pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -31,9 +29,35 @@ buildPythonPackage rec {
 
   buildInputs = [ python ];
 
-  propagatedBuildInputs = [ ansible enrich pyyaml rich ruamel-yaml wcmatch tenacity flaky yamllint ];
+  propagatedBuildInputs = [
+    ansible
+    enrich
+    flaky
+    pyyaml
+    rich
+    ruamel-yaml
+    tenacity
+    wcmatch
+    yamllint
+  ];
+
+  checkInputs = [
+    pytest-xdist
+    pytestCheckHook
+  ];
+
+  pytestFlagsArray = [
+    "--numprocesses" "auto"
+  ];
 
-  checkInputs = [ pytestCheckHook pytest-xdist git ];
+  postPatch = ''
+    # fixes test_get_yaml_files_umlaut and test_run_inside_role_dir
+    substituteInPlace src/ansiblelint/file_utils.py \
+      --replace 'os.path.join(root, name)' 'os.path.normpath(os.path.join(root, name))'
+    # fixes test_custom_kinds
+    substituteInPlace src/ansiblelint/file_utils.py \
+      --replace "if name.endswith('.yaml') or name.endswith('.yml')" ""
```

Comment:
Can you open up issues upstream and reference them here? I think that is all that is left to do.

Reply:
Yes, I'll do that